src/rpc/handlers/AccountTx.h

```diff
@@ -131,6 +148,23 @@ class AccountTxHandler {
     process(Input input, Context const& ctx) const;
 
 private:
+    static std::unordered_set<std::string> const&
```

Comment:
Would this not be useful as a public utility of sorts? That way not only we could reuse it if we need to but also we could unit-test it outside of the handler entirely. WDYT?